cluster-provision/ocp-cnao/4.4/provision.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -x
```

Comment:
Do we need to copy paste the whole thing? Cannot we share some code?

Reply:
Yep, just like the other providers, at the end most of it are just parameters to gocli, also it does not change that much.